picard/file.py

```diff
@@ -229,7 +230,7 @@ def _preserve_times(self, filename, func):
             # https://docs.python.org/3/library/os.html#os.utime
             # Since Python 3.3, ns parameter is available
             # The best way to preserve exact times is to use the st_atime_ns and st_mtime_ns
-            # fields from the os.stat() result object with the ns parameter to utime.
+            #  fields from the os.stat() result object with the ns parameter to utime.
```

Comment:
This change is unrelated.

Reply:
ugh, I turned on some of the auto formatting in my editor, and it got a little over zealous.  Same for next few.